pkgs/development/libraries/libappindicator/default.nix

```diff
@@ -54,11 +55,17 @@ stdenv.mkDerivation {
     "localstatedir=\${TMPDIR}"
   ];
 
+  passthru.tests.pkg-config = testers.testMetaPkgConfig finalAttrs.finalPackage;
+
   meta = with lib; {
     description = "A library to allow applications to export a menu into the Unity Menu bar";
     homepage = "https://launchpad.net/libappindicator";
     license = with licenses; [ lgpl21 lgpl3 ];
+    pkgConfigModules = {
+      "2" = [ "appindicator-0.1" ];
+      "3" = [ "appindicator3-0.1" ];
+    }.${gtkVersion};
```

Comment:
We nowhere assert if gtkVersion is 2 or 3. It could also be 4 and then we run into a syntax error which should never happen.

Reply:
See the `if gtkVersion == "2"` above. That is not correct if there are other options.

Reply:
I fixed that in a second commit.